sbt-play-ebean/src/main/scala/play/ebean/sbt/PlayEbean.scala

```diff
@@ -25,7 +40,24 @@ object PlayEbean extends AutoPlugin {
 
   override def trigger = noTrigger
 
-  override def projectSettings = inConfig(Compile)(scopedSettings) ++ unscopedSettings
+  override def projectSettings = {
+    val querySettings = Seq(
+      playEbeanQueryGenerate := false,
+      playEbeanQueryEnhance := false,
+      playEbeanQueryDestDirectory := "app",
```

Comment:
This might be useful, after reading the previous comments I thought perhaps the way to go was finders -> source control and regular beans -> managed sources.
Ideally this could be done in the same step however it probably requires modification of the existing generation code.

If I ran it twice I'd also have to purge the generated query classes.
I think it's probably worth either making a modification to the other project or just ignoring it as you suggested. The user can always manually copy it to their source control although they'd have to disable the generation afterwards possibly.